packages/x-components/src/components/result/base-result-image.vue

```diff
@@ -41,7 +41,7 @@
 <script lang="ts">
   import { Result } from '@empathyco/x-types';
```

Comment:
About adding a test with Cypress component testing, the problem with that framework is that it is intended to test the component from a user perspective, so if you want to change props, you have to build a system that allows the user to change them. It is more an integration test than pure unit testing. 
For example, in this case I imagine you can write a test combining the component with the ResultProvider/ResultSelector components. Which will give you 100% real guarantee that these components work properly together, which is really nice.

Reply:
Check the test I added and if it fits with the requirements